core/domain/exp_domain_test.py

```diff
@@ -2904,7 +3364,7 @@ class SchemaMigrationUnitTests(test_utils.GenericTestBase):
         param_changes: []
         refresher_exploration_id: null
       hints: []
-      id: TextInput
+      id: FractionInput
```

Comment:
Why we are changing the interaction here? (I don't think we had Fraction interaction around v21 of exploration.)

Reply:
`_convert_v21_dict_to_v22_dict` calls `_convert_states_v16_dict_to_v17_dict` and we are using FractionInput in `_convert_states_v16_dict_to_v17_dict`. See here:
https://github.com/oppia/oppia/blob/ea2a0db572c96c65543bf6faccebf0dde033c1e9/core/domain/exp_domain.py#L1806